book/http.md

```diff
@@ -927,11 +927,11 @@ make it so that, if your browser is started without a URL being given,
 some specific file on your computer is opened. You can use that file
 for quick testing.
 
-*Alternate encodings:* read the encoding from the `Content-Type`
+*Alternate encodings:* Read the encoding from the `Content-Type`
 header instead of always using `utf8`. Test it on a real site that
 doesn't use `utf8`, like `google.com`.
 
-*data:* Yet another scheme is *data*, which
+*Data:* Yet another scheme is *data*, which
```

Comment:
I thing this is maybe wrong and instead we meant data in backticks as in the scheme name.